src/Toolkit/Toolkit.UWP/UI/Controls/FloorFilter/FloorFilter.Includes.xaml

```diff
@@ -0,0 +1,8 @@
+﻿<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:controls="using:Esri.ArcGISRuntime.Toolkit.UI.Controls"
+    xmlns:internal="using:Esri.ArcGISRuntime.Toolkit.Internal"
+    xmlns:toolkit="using:Esri.ArcGISRuntime.Toolkit.UI.Controls">
+    <FontFamily x:Key="CalciteIconFont">ms-appx:///Esri.ArcGISRuntime.Toolkit.UWP/Assets/calcite-ui-icons-24.ttf#calcite-ui-icons-24</FontFamily>
```

Comment:
Include this once in generic.xaml instead, and let all styles be able to reuse it.